po/be.po

```diff
@@ -3405,7 +3405,7 @@ msgstr ""
 msgid "Toggle Insert/Overwrite mode"
```

Comment:
> Yes, I tried to restore the not changed translation files to the original state but that did not work.

It might be a little tricky because the files were updated in master and you merged that, so you can't simply revert.  I restored the version from master in https://github.com/b4n/geany-plugins/tree/LarsGit223/workbench%2Brestored_trans (commit add14f7117926b93ae02e390f1a079863203d370).

> Maybe I should place a new fresh PR is all points are sorted out to keep the other translation files unchanged?

As you like, I guess ideally you'd rebase your changes on top of master and force-push to this PR once it's all sorted out.